core/services/flux_monitor_fetchers.go

```diff
@@ -113,15 +108,11 @@ type medianFetcher struct {
 func newMedianFetcherFromURLs(
 	timeout time.Duration,
 	requestData string,
-	priceURLs ...string,
+	priceURLs []*url.URL,
```

Comment:
might be best to keep the ...*url.URL as syntax sugar

Reply:
I paired on this part with @j16r and he had recommended changing... not sure what our convention has been here?

Reply:
Given that in every case we use this we have to use the splat operator to unroll an array, it seems that we are not using it - so I say YAGNI - get rid of it!